src/libs/karm-net/http/http.h

```diff
@@ -169,11 +170,20 @@ struct Version {
         v.minor = try$(atou(s));
         return Ok(v);
     }
+
+    Res<> unparse(Io::TextWriter& w) {
+        try$(Io::format(w, "HTTP/{}.{}", major, minor));
+        return Ok();
+    }
 };
 
 struct Header {
     Map<Str, Str> headers;
```

Comment:
Since the type is no longer just used for parsing, we might want to change that to Map<String, String>